maps/_ihme-explorer/src/utils/config.js

```diff
@@ -0,0 +1,101 @@
+import * as _ from 'underscore';
+
+export const loadConfig = (config, setModelVersion, setDates, setActiveDate) => {
+    // Set model version
+    setModelVersion(config['ihme_version']);
+
+    // Handle dates
+    let dates = config['dates'],
+        sortedDates = Array.from(new Set(dates)).sort();
+    setDates(sortedDates);
+
+    const today = new Date().toISOString().slice(0, 10),
+    todayIndex = sortedDates.indexOf(today);
+
+    setActiveDate(todayIndex !== -1 ? todayIndex : sortedDates.length - 1);
+
+    // Handle aggregation type properties
+    let aggTypeConfig = config['aggregations'];
+    _.pairs(aggTypeConfig).map(kv => {
+        let aggConfig = aggregationTypes[kv[0]];
+        _.pairs(kv[1]).map(kv2 => {
+            aggConfig[kv2[0]] = kv2[1];
+        });
+    });
+
+    return config;
+};
+
+export const aggregationTypes = {
+    'country': {
+        label: 'National'
+    },
+    'region': {
+        label: 'Regional',
+        default: true
+    }
+};
+
+// Keys match IHME columns
+export const indicators = {
```

Comment:
I think we should visually reorder these indicators to match up more closely with what they show on https://covid19.healthdata.org/ vs what's given in their data csv/info sheet.

Suggested reordering:
`deaths` <--also suggest making this the indicator to show by default
`admis`
`newICU`
`allbed`
`ICUbed`
`InvVen`
`bedover`
`icuover`
`totdea`